src/tests/Interop/Swift/SwiftErrorHandling/SwiftErrorHandling.cs

```diff
@@ -0,0 +1,69 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Runtime.InteropServices.Swift;
+using System.Text;
+using Xunit;
+
+public class ErrorHandlingTests
+{
+    private const string SwiftLib = "libSwiftErrorHandling.dylib";
+
+    [DllImport(SwiftLib, EntryPoint = "$s18SwiftErrorHandling05setMyB7Message7message6lengthySPys6UInt16VG_SitF")]
+    public static extern void SetErrorMessage(char[] message, int length);
```

Comment:
```suggestion
    [DllImport(SwiftLib, EntryPoint = "$s18SwiftErrorHandling05setMyB7Message7message6lengthySPys6UInt16VG_SitF", CharSet = CharSet.Unicode)]
    public static extern void SetErrorMessage(string message, int length);
```

Reply:
Thank you, that is much cleaner solution.